### PR TITLE
Fix/list view repairs

### PR DESCRIPTION
**Pull request by paulmskim**
🎫 https://central.tri.be/issues/128344

### CODE REVIEW

**Comment by juanfra** [comment]
@paulmskim are you waiting for the ticket they mentioned yesterday during scrum?

**Comment by paulmskim** [comment]
@juanfra that and anything else that comes back from matt.

**Comment by juanfra** [comment]
@paulmskim here: https://central.tri.be/issues/128344

File: src/views/v2/list/nav.php [hunk 1]
```diff
@@ -12,10 +12,15 @@
  * @version TBD
  *
  */
+
+if ( ! tribe_has_previous_event() || ! tribe_has_next_event() ) {
```
Comment (juanfra) [inline src/views/v2/list/nav.php]: Missed this one. The `||` should be an `&&`

Comment (paulmskim) [reply src/views/v2/list/nav.php]: ah yes, thanks! will fix.

**Pull request merged**